src/Microsoft.Azure.WebJobs.Host/JobHostBlobsConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Represents configuration for <see cref="BlobTriggerAttribute"/>.
+    /// </summary>
+    public class JobHostBlobsConfiguration
```

Comment:
Likely there will be additional blob scan knobs we want to expose in the future as well. I do remember this coming up before but we didn't have a way previously.